lms/djangoapps/shoppingcart/models.py

```diff
@@ -1746,6 +1746,8 @@ def purchased_callback(self):
         self.course_enrollment.activate()
 
     def additional_instruction_text(self):
+        if not self.course_enrollment.is_verified_enrollment():
+            return ""
```

Comment:
Do we not offer refunds for no-ID professional courses? Returning an empty string here won't add a refund reminder to the payment confirmation email. It would be better to set the verification reminder below to the empty string if the current enrollment isn't verified; this will still add the refund reminder to the email.